### PR TITLE
Move Astronomer helm values to Product Specific TF repos

### PR DESCRIPTION
**Pull request by kaxil**
By default we don't want to override any Helm values. We would override it based on the Product they use - Enterprise or Cloud.

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -18,11 +13,6 @@ variable "tls_key" {
   description = "The private key corresponding to the signed certificate tls_cert."
 }
 
-variable "base_domain" {
-  type        = string
-  description = "<var.deployment_id>.<var.route53_domain>"
-}
-
```
Comment (ianstanton) [inline variables.tf]: Will these removed variables be hard-coded in our `astronomer_helm_values` variable?

File: astronomer.tf [hunk 2]
```diff
@@ -57,6 +57,6 @@ resource "helm_release" "astronomer" {
   repository = data.helm_repository.astronomer_repo.name
   namespace  = var.astronomer_namespace
   wait       = true
-  values = [local.astronomer_values]
+  values = [var.astronomer_helm_values]
 }
 */
```
Comment (ianstanton) [inline astronomer.tf]: This looks good to me

**Review by sjmiller609 — APPROVED** [review]
Looks good

File: variables.tf (hunk 1 above)
Comment (kaxil) [reply variables.tf]: Nope, they will still be in TF variable but in the other TF module -> https://github.com/astronomer/terraform-google-astronomer-cloud/pull/8/files

**Pull request merged**